tests/integration/ceph_upgrade_test.go

```diff
@@ -170,6 +170,17 @@ func (s *UpgradeSuite) testUpgrade(useHelm bool, initialCephVersion v1.CephVersi
 	s.verifyFilesAfterUpgrade(newFile, rbdFilesToRead, cephfsFilesToRead)
 	logger.Infof("Verified upgrade from reef to squid")
```

Comment:
We also need an upgrade test such as upgrade-from-tentacle-stable-to-tentacle-devel, similar to what was added for squid in #14454.

Reply:
added

Reply:
Looks like the github workflow update is also needed to trigger that test.